web/elm/src/Assets.elm

```diff
@@ -135,12 +136,18 @@ toPath asset =
             in
             basePath ++ [ "baseline-visibility" ++ imageName ++ ".svg" ]
 
-        FavoritedToggleIcon isFavorited ->
+        FavoritedToggleIcon { isFavorited, isHovered, isSideBar } ->
             let
                 imageName =
                     if isFavorited then
                         "-filled"
 
+                    else if isSideBar && isHovered then
+                        "-unfilled-bright"
```

Comment:
Is it possible the colours are still a bit off on this icon? Hovering an unfavourited pipeline in the sidebar, I see:

<img width="393" alt="Screen Shot 2020-11-03 at 9 20 34 AM" src="https://user-images.githubusercontent.com/26583442/97997124-a1c18600-1db6-11eb-8e03-7c21d99e1326.png">

The text appears to be #E2E2E2, but the icon appears to be #C6C6C6

Reply:
VERY good eye, I'll defer to @matthewpereira (since in the [mocks](https://www.figma.com/file/fuoFSVcTGptiS21P2POA2XbN/Concourse-UI-kit-and-Experiments?node-id=1509%3A55) it has the #e2e2e2 / #c6c6c6 difference on the sidebar)

Reply:
I think that must be an error on my behalf.

I think the goal is:
Default colour for icons and text (eg, pipeline icon, pipeline name, favorite icon) is #C6C6C6
Hover colour for icons and text is #FFFFFF
Active/selected colour for icons and text is #E2E2E2, with a #151515 background

Default colour for team icons and text is #E2E2E2 in order to give them a little more visual prominence

Does that make sense?

Reply:
OK! 

So default/unhovered colours for the sidebar:
- team icon/text -> #E2E2E2
- pipeline icon/text -> #C6C6C6

Reply:
@matthewpereira i just pushed an update, I'd love another look at it to see if it's what you specified above :)